site/content/entry/citra-progress-report-2018-Q1.md

```diff
@@ -0,0 +1,247 @@
+added in Citra. Some notable ones are multiplayer, hardware shader improvements (post GLvtx),
+logging rewrite and the highly coveted camera support. Our build generating environment
+was vastly improved as well. Apart from these, we have many more minor features,
+improvements and bug fixes.
+
+***Now let’s see these in detail:***
+
+## Citra-qt : Multiplayer support by [jroweboy](https://github.com/jroweboy), [B3N30](https://github.com/B3N30), [Subv](https://github.com/subv) and [JayFoxRox](https://github.com/jayfoxrox)
+
+Citra has been able to run many games at playable speeds for a while now. But it
+was always lacking something which the 3DS had. The ability to play with friends.
+So, to make it a reality our developers [jroweboy](https://github.com/jroweboy),
+[B3N30](https://github.com/B3N30), [Subv](https://github.com/subv) and
+[JayFoxRox](https://github.com/jayfoxrox) worked tirelessly for months to reverse
+engineer the working of 3DS local wireless system. Citra doesn't support or emulate
+online play or download play (***yet!*** ). But what it does, is that it now emulates
+_local wireless multiplayer_ over the internet. This allows you to play your most
+favorite games with your virtual friends across the world.
+
+**Note**: Although multiplayer is supported, some games might have known/unknown issues.
+Since our servers are located at various locations, latency will play a major role in
+your gameplay experience. Please use our [support forums](https://community.citra-emu.org/),
+[IRC](http://webchat.freenode.net/?channels=citra) or [discord](https://citra-emu.org/discord/)
+to inform us of any issues with multiplayer.
+
+#### **Citra multiplayer features**:
+* We now have multiple servers hosted by our very own [Flame Sage](https://github.com/chris062689),
+and many other community members at various locations around the globe, so that
+players can meet new players online.
+* Rooms which can hold up to 16 players can be created and even be secured with a
+password.
+* You can now also set your preferred game.
+* Since we now have many number of servers, you can also filter your servers to
+suit your needs.
+
+{{< figure src="/images/entry/citra-progress-report-2018-q1/Server_list.png"
+    title="Sooo many servers to choose from!" >}}
+
+Since its initial implementation in the Canary, developers have listened to
+the community and tried very hard to fix bugs with many games which weren't
+compatible at that time and also have added other features to make the multiplayer
+experience richer. Currently we are proud to announce that multiplayer works in
+the following list of titles.
+
+#### **List of multiplayer compatible titles**:
+* Dragon Quest Monsters : Joker 3 Professional
+* Luigi's Mansion: Dark Moon
+* Mario Party: The Top 100
+* Monster Hunter 3G & 3 Ultimate
+* Monster Hunter 4, 4G & 4 Ultimate
+* Monster Hunter XX
+* Monster Hunter Generations
+* New Super Mario Bros. 2
+* Pokemon Omega Ruby / Alpha Sapphire
```

Comment:
Pok**é**mon
applies to any other instance as well

Reply:
fixed